api/v1/installation_types.go

```diff
@@ -604,6 +604,10 @@ type InstallationStatus struct {
 	// Computed is the final installation including overlaid resources.
 	// +optional
 	Computed *InstallationSpec `json:"computed,omitempty"`
+
+	// Conditions represents the latest observed set of conditions for this component. A component may be one or more of
+	// Available, Progressing, or Degraded.
+	Conditions []TigeraStatusCondition `json:"conditions,omitempty"`
```

Comment:
I think the type of the object should be []Condition using the metav1 import: "k8s.io/apimachinery/pkg/apis/meta/v1"

Reply:
I'm sorry. That is not compatible. Please disregard.